src/engine/physics/physics.game.ts

```diff
@@ -130,6 +121,18 @@ export const PhysicsSystem = (state: GameState) => {
     }
   }
 
+  // remove rigidbody from physics world
+  const exited = exitedPhysicsQuery(world);
```

Comment:
Does this fix our issue with rigid body disposal? Should it close https://github.com/matrix-org/thirdroom/issues/147 ?

Reply:
i thought it did, but it did not unfortunately :(

Reply:
i added notes to #147